data_labeling_examples/bulk_labeling_java/README.md

```diff
@@ -88,7 +88,40 @@ Conditions -
     TRAINING_DATASET_ID (Required only for training a new model)
     
 ```
+**3. Remove labels of records in Data Labeling Service**
 
+**RemoveLabelScript:** This script takes REMOVE_LABEL_PREFIX as input and remove the labels from records which are matching with REMOVE_LABEL_PREFIX.
+REMOVE_LABEL_PREFIX will be a label name, or label name prefix or '*'. 
+
+If '*' is given as REMOVE_LABEL_PREFIX then it will remove all labels from all records.
+
+```
+Consider a dataset having following records:
+cat1.jpeg, cat2.jpeg, dog1.jpeg, dog2.jpeg
+Labels in dataset: dog, pup, cat, kitten
+    cat1.jpeg will be labeled with cat label
+    cat2.jpeg will be labeled with cat and kitten labels
+    dog1.png will be labeled with dog label
+    dog2.png will be labeled with dog and pup labels
+    
+1. If REMOVE_LABEL_PREFIX = 'c' then it will remove label 'cat' from all labeled records. Dataset will be as folows :
+    cat1.jpeg -> unlabeled
+    cat2.jpeg will be labeled with kitten labels
+    dog1.png will be labeled with dog label
+    dog2.png will be labeled with dog and pup labels
+    
+2. If REMOVE_LABEL_PREFIX = 'd' then it will remove label 'dog' from all labeled records. Dataset will be as folows :
+    cat1.jpeg will be labeled with cat label
+    cat2.jpeg will be labeled with kitten labels
+    dog1.png -> unlabeled
+    dog2.png will be labeled with dog and pup labels
+
+3. If REMOVE_LABEL_PREFIX = '*' then it will remove all labels from all labeled records. Dataset will be as folows :
```

Comment:
This is a clean method to allow users to delete all annotations!

Reply:
yes, it will delete all annotations